meson.build

```diff
@@ -741,7 +741,9 @@ substs.set('USERS_GID', users_gid)
 conf.set10('ENABLE_ADM_GROUP', get_option('adm-group'))
 conf.set10('ENABLE_WHEEL_GROUP', get_option('wheel-group'))
 
-substs.set('DEV_KVM_MODE', get_option('dev-kvm-mode'))
+dev_kvm_mode = get_option('dev-kvm-mode')
+substs.set('DEV_KVM_MODE', dev_kvm_mode)
+conf.set10('DEV_KVM_UACCESS', dev_kvm_mode != '0666')
```

Comment:
nitpick: Does an integer comparison work here?

```python
conf.set10('DEV_KVM_UACCESS', int(dev_kvm_mode) != 0o0666)
```

And should this maybe check whether the "other" bits are set, regardless of which of them are?

```python
conf.set10('DEV_KVM_UACCESS', (int(dev_kvm_mode) & 0o7) != 0)
```

(This last one probably needs a comment to clarify what that does...)

Reply:
Thanks for your review.
What would be the benefit of doing an integer comparison? Tbh, I find the version with the string comparison a bit more readable. But I can change it if requested.